### DISCUSSION TITLE
multi domain handling

### DISCUSSION THREAD

**Discussion by gchokeen** [nuxt/nuxt#16537] [Questions]
I have a nuxt application thats resolving multiple domain address, for example `site-a.local` `site-b.local` `site-c.local`. Each site will have different logo, some content difference, theme color, layout difference, I'm doing everything using numerical site id like `2,3,4`. So I unable to find the perfect way to assign site id based on the domains. 

Note: 
1. It's the server side rendering application
2. I don't want to show the numerical site id in the url anywhere
3. I use nginx webserver and reverse proxying to nuxt application

**Comment by paulvonber** [comment]
You can use x-forwarded-host header and identify your site accordingly to your domain name.

**Comment by TheAlexLichter** [comment]
See https://github.com/nuxt/framework/discussions/6057
